### ISSUE TITLE
Changes in 1.1.4 cause issues with browserify/babelify

### ISSUE THREAD

**Issue by sstone1** [Starcounter-Jack/JSON-Patch#140]
The changes in 1.1.4 cause issues with a combination of browserify/babelify.
All the functions go missing and cannot be called in the resulting file.

I've built a small reproduce and posted it here: https://github.com/sstone1/JSON-Patch-issue
To recreate after a `git clone` and `npm install`, run the following command:

```
$ npm run browserify && node generated.js

> fast-json-patch-issue@1.0.0 browserify /Users/sstone1/git/fast-json-patch-issue
> browserify ./index.js -s generated -d -t [ babelify --presets [ es2015 ] --global ] > ./generated.js

jsonpatch.compare = undefined
/Users/sstone1/git/fast-json-patch-issue/generated.js:6
console.log(JSON.stringify(jsonpatch.compare({ a: 'b' }, { c: 'd' })));
                                     ^

TypeError: jsonpatch.compare is not a function
    at Object.1.fast-json-patch (/Users/sstone1/git/fast-json-patch-issue/generated.js:6:38)
    at s (/Users/sstone1/git/fast-json-patch-issue/generated.js:1:685)
    at e (/Users/sstone1/git/fast-json-patch-issue/generated.js:1:856)
    at /Users/sstone1/git/fast-json-patch-issue/generated.js:1:874
    at a (/Users/sstone1/git/fast-json-patch-issue/generated.js:1:150)
    at Object.<anonymous> (/Users/sstone1/git/fast-json-patch-issue/generated.js:1:387)
    at Module._compile (module.js:556:32)
    at Object.Module._extensions..js (module.js:565:10)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
```

With the previous version of this module, the compare function is available:

```
$ npm install fast-json-patch@1.1.3
...
$ npm run browserify && node generated.js

> fast-json-patch-issue@1.0.0 browserify /Users/sstone1/git/fast-json-patch-issue
> browserify ./index.js -s generated -d -t [ babelify --presets [ es2015 ] --global ] > ./generated.js

jsonpatch.compare = function compare(tree1, tree2) {
        var patches = [];
        _generate(tree1, tree2, patches, '');
        return patches;
    }
[{"op":"remove","path":"/a"},{"op":"add","path":"/c","value":"d"}]
```

**Comment by philwhln** [comment]
I'm hitting this too.

**Comment by tomalec** [comment]
Thanks @sstone1 for great issue report.

If we will not fix it within a week, we will release 1.1.5 with reverted changes.

**Comment by JSteunou** [comment]
@tomalec what's the status on this? We also have this issue with webpack. I'm afraid all CommonJS import are broken.
